### PR TITLE
Add profile image to exposed api

### PR DESCRIPTION
**Pull request by msfci**

### CODE REVIEW

File: sm-shop/src/main/java/com/salesmanager/shop/store/api/v1/customer/AuthenticateCustomerApi.java [hunk 1]
```diff
@@ -40,6 +35,7 @@
 import io.swagger.annotations.ApiOperation;
 import io.swagger.annotations.SwaggerDefinition;
 import io.swagger.annotations.Tag;
+import org.springframework.web.multipart.MultipartFile;
```
Comment (gemy21ce) [inline sm-shop/src/main/java/com/salesmanager/shop/store/api/v1/customer/AuthenticateCustomerApi.java:38]: unused import?

File: sm-shop/src/main/java/com/salesmanager/shop/store/controller/customer/facade/CustomerFacadeImpl.java [hunk 2]
```diff
@@ -1071,4 +1056,68 @@ public void changePassword(Customer customer, String newPassword) {
...
+        throw new ConversionRuntimeException(e);
+      }
+
+      String password = customer.getPassword();
+      if (StringUtils.isBlank(password) && generateRandomPassword) {
+        password = UserReset.generateRandomString();
+        customer.setPassword(password);
+      }
+
+      Optional.ofNullable(customer.getProfilePhoto()).ifPresent(profilePhotoFile -> {
```
Comment (gemy21ce) [inline sm-shop/src/main/java/com/salesmanager/shop/store/controller/customer/facade/CustomerFacadeImpl.java:1105]: isn't it simpler to use if null?

File: sm-shop/src/main/java/com/salesmanager/shop/store/api/v1/customer/AuthenticateCustomerApi.java (hunk 1 above)
Comment (AhmedNashatAli) [reply sm-shop/src/main/java/com/salesmanager/shop/store/api/v1/customer/AuthenticateCustomerApi.java:38]: nice catch

File: sm-shop/src/main/java/com/salesmanager/shop/store/controller/customer/facade/CustomerFacadeImpl.java (hunk 2 above)
Comment (AhmedNashatAli) [reply sm-shop/src/main/java/com/salesmanager/shop/store/controller/customer/facade/CustomerFacadeImpl.java:1105]: Maybe you have a point here but let's have a concept of using the new features of java and that was my recommendation

File: sm-shop/src/main/java/com/salesmanager/shop/store/api/v1/customer/AuthenticateCustomerApi.java (hunk 1 above)
Comment (msfci) [reply sm-shop/src/main/java/com/salesmanager/shop/store/api/v1/customer/AuthenticateCustomerApi.java:38]: Which one?

**Pull request merged**